arch/s390/slide_hash_vx.c

```
@@ -0,0 +1,35 @@
+        uv8hi value, result;
+
+        value = vec_xl(0, p);
+        result = vec_sub(value, vec_min(value, vmx_wsize));
+        vec_xst(result, 0, p);
+
+        p += 8;
+        entries -= 8;
+    } while (entries > 0);
+}
+
+void Z_INTERNAL slide_hash_vx(deflate_state *s) {
```

Cppcheck (2.20.0)

[style] The function 'slide_hash_vx' is never used.

(unusedFunction)